tests/NavTest.php

```diff
@@ -119,4 +119,33 @@ public function testEmptyItems()
 
         $this->assertEqualsWithoutLE($expected, $out);
     }
+
+    /**
+     * @see https://github.com/yiisoft/yii2-bootstrap/issues/162
+     */
+    public function testExplicitActive()
+    {
+        Nav::$counter = 0;
+        $out = Nav::widget([
+            'activateItems' => false,
+            'items' => [
+                [
+                    'label' => 'Item1',
+                    //'url' => ['some/route1'],
```

Comment:
Shouldnt we strive to consistent naming with respect to readability. So far, I've seen:
* $result
* $out
* $actual
* etc

Shouldnt we choose for 1?